.gitignore

```diff
@@ -33,6 +33,7 @@ bld/
 [Oo]bj/
 [Ll]og/
 [Ll]ogs/
+build/
```

Comment:
This is not needed. :) Users might name that directory anything, so it is impossible to guess in advance (a common one is `builddir/`, and many people use multiple build directories for e.g. different buildtypes). On the other hand, Meson knows when it is created that it is a build directory, and that nothing inside it should ever be checked into git.

So as a result, Meson does the logical, convenient thing, and creates a `.gitignore` inside the build directory, which ignores `*` to completely prevent any files in there from getting checked into git. You don't need to do anything, Meson just does the right thing for you out of the box.

Reply:
Ah, that makes sense. Appreciate the reminder.